apps/condo/domains/property/components/panels/ResidentPanel.tsx

```diff
@@ -0,0 +1,56 @@
+import { useIntl } from '@core/next/intl'
+import { BasicEmptyListView } from '@condo/domains/common/components/EmptyListView'
+import { Col, Row, Typography, Space, Tooltip } from 'antd'
+import React from 'react'
+import { Button } from '@condo/domains/common/components/Button'
+
+const EmptyResidentBlock: React.FC = () => {
+    const intl = useIntl()
+    const EmptyResidentsTitle = intl.formatMessage({ id: 'pages.condo.property.form.EmptyResidents.header' })
+    const EmptyResidentsMessage = intl.formatMessage({ id: 'pages.condo.property.form.EmptyResidents.text' })
+    const UploadFile = intl.formatMessage({ id: 'UploadFile' })
+    const NotImplementedYedMessage = intl.formatMessage({ id: 'NotImplementedYed' })
```

Comment:
`NotImplementedYed`, `Yed`...
This is my mistake, can you, please, rename all `Yed` occurrences for `Yet`?)

Reply:
Done